config/config.go

```diff
@@ -10,6 +10,7 @@ import (
 // Configuration structure which hold information for configuring the import API
 type Configuration struct {
 	BindAddr                 string        `envconfig:"BIND_ADDR"`
+	FeatureDetachDataset     bool          `envconfig:"FEATURE_DETACH_DATASET"`
```

Comment:
Given that our other feature flag in this app is `EnablePrivateEnpoints` is it worth being consistent with that format and using the word Enable rather than Feature? (may be worth a team chat to see if we want to set a standard around feature flags as we start using them more)

Reply:
good point. Didn't think of that as a feature flag, too used to it now :)